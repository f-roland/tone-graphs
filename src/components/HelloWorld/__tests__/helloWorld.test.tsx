import React from "react";
import { HelloWorld } from "..";
import renderer from "react-test-renderer";

describe("it renders correctly", () => {
  it("renders correctly", () => {
    const component = renderer.create(<HelloWorld />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
