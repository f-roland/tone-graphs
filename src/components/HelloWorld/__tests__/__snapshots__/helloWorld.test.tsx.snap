// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 16,
      }
    }
  >
    Hello 👋
  </Text>
  <Text
    style={
      Object {
        "fontSize": 16,
      }
    }
  >
    Happi Karaoke v2 is coming soon ! 🚀 🎸 🎹 🥁 
  </Text>
</View>
`;
